### ISSUE TITLE
Enable "eslint.workingDirectories" configuration option as vscode

### ISSUE THREAD

**Issue by ravisumit33** [neoclide/coc-eslint#80]
It will be better to use `eslint.workingDirectories` option if you have multiple projects in your workspace.

My project directory structure is:
```
├── frontend
│   └── .eslintrc
│   └── node_modules
│   └── package.json
├── backend
│   └── requirements.txt
├── .git
├── README.md
```

My eslintrc is :
```
{
    "extends": "react-app",
    "parser": "babel-eslint",
    "rules": {
        "eol-last": ["error", "always"]
    }
}
```

Without `eslint.workingDirectories` option eslint doesn't work. Even in vs-code we need to use this option as:
```
"eslint.workingDirectories": [
    { "mode" : "auto" }
]
```
Please consider enabling this option.

**Comment by ravisumit33** [comment]
@chemzqm Any thoughts on this?

**Comment by sangdth** [comment]
Currently I'm working on a mono repo project, in which this option is really important to make ESLint works correctly in sub package.

**Comment by chemzqm** [comment]
Fixed on 1.4.0.
